### PR TITLE
Allow running blivet without Python SELinux module

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: blivet/util.py [hunk 1]
```diff
@@ -450,6 +456,8 @@ def get_cow_sysfs_path(dev_path, dev_sysfsPath):
 
 def match_path_context(path):
```
Comment (vojtechtrefny) [inline blivet/util.py:457]: All calls to these function are "protected" by checking the SELinux flag first, so we should never raise the `RuntimeError` below, but these are still "public" functions so it is probably good idea to add the check just to be safe.

File: blivet/flags.py [hunk 2]
```diff
@@ -40,7 +39,13 @@ def __init__(self):
         #
         # enable/disable functionality
         #
-        self.selinux = selinux.is_selinux_enabled()
+        try:
+            import selinux
+        except ImportError:
+            self.selinux = False
```
Comment (vojtechtrefny) [inline blivet/flags.py:45]: We might want to log this for situations where SELinux is enabled but `python3-libselinux` is not installed. Thoughts?

**Review by japokorn — APPROVED** [review]
Looks good to me.

**Review by dwlehman — COMMENTED** [review]
Looks good to me, but I wonder if it would be more consistent to use `tasks`/`availability` for that part of it.

**Pull request merged**